composer.json

```diff
@@ -17,10 +17,11 @@
     "require": {
         "php": ">=5.4.0",
         "illuminate/view": "5.*",
-        "zordius/lightncandy": "1.0.*"
+        "zordius/lightncandy": "1.1.*",
```

Comment:
Thanks 👍 
Sorry for being pedantic… but would you mind removing the trailing comma?
I will then put up a new release immediately.